src/Shared/NativeMethodsShared.cs

```diff
@@ -137,6 +137,39 @@ private enum eDesiredAccess : int
             PROCESS_ALL_ACCESS = SYNCHRONIZE | 0xFFF
         }
 
+        internal enum LOGICAL_PROCESSOR_RELATIONSHIP
+		{
+			RelationProcessorCore,
+			RelationNumaNode,
+			RelationCache,
+			RelationProcessorPackage,
+			RelationGroup,
+			RelationAll = 0xffff
+		}
+
+        internal struct SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX
+        {
+            public LOGICAL_PROCESSOR_RELATIONSHIP Relationship;
+            public uint Size;
+            public GROUP_RELATIONSHIP Group;
+        }
+
+        internal unsafe struct GROUP_RELATIONSHIP
+        {
+            private byte MaximumGroupCount;
```

Comment:
nit: This is defined as
```C++
WORD                 MaximumGroupCount;
```
in the unmanaged world so should be an `ushort` here.